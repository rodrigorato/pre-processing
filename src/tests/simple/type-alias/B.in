alias Cache = ConcurrentSkipListMap<String,List<Map<String,Object>>>;
public class Foo {
	public static Cache mergeCaches(Cache a, Cache b) {
		Cache vary = new Cache();
	}
}