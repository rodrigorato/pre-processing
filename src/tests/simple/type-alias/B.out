
public class Foo {
	public static ConcurrentSkipListMap<String,List<Map<String,Object>>> mergeCaches(ConcurrentSkipListMap<String,List<Map<String,Object>>> a, ConcurrentSkipListMap<String,List<Map<String,Object>>> b) {
		ConcurrentSkipListMap<String,List<Map<String,Object>>> vary = new ConcurrentSkipListMap<String,List<Map<String,Object>>>();
	}
}